Procurement/ViewModel/LoginWindowViewModel.cs

```diff
@@ -189,6 +189,12 @@ private IEnumerable<Item> LoadItems(bool offline, IEnumerable<Character> chars)
                 if (character.League == "Void")
                     continue;
 
+                if(character.Expired == true)
+                {
+                    statusController.DisplayMessage("\nSkipping character " + character.Name + " because the characters name has expired.\n");
```

Comment:
Up higher in the same file there are a bunch of \n instead of Environment.NewLine, just FYI.